test/integration_test/storkctl_migrationschedule_test.go

```diff
@@ -196,6 +201,20 @@ func ValidateMigrationSchedule(t *testing.T, migrationSchedule *storkv1.Migratio
 	require.Equal(t, migrationSchedule.Spec.Template.Spec.StartApplications, actualMigrationSchedule.Spec.Template.Spec.StartApplications, "MigrationSchedule StartApplications mismatch")
 	require.Equal(t, migrationSchedule.Spec.Template.Spec.TransformSpecs, actualMigrationSchedule.Spec.Template.Spec.TransformSpecs, "MigrationSchedule TransformSpecs mismatch")
 	require.Equal(t, migrationSchedule.Spec.Template.Spec.IncludeNetworkPolicyWithCIDR, actualMigrationSchedule.Spec.Template.Spec.IncludeNetworkPolicyWithCIDR, "MigrationSchedule IncludeNetworkPolicyWithCIDR mismatch")
+
+	if validateSchedulePolicyInterval != "" {
+		//we will validate the created schedule policy has correct interval minutes value as well
+		expectedInterval, err := strconv.Atoi(validateSchedulePolicyInterval)
+		if err != nil {
+			return err
+		}
+		actualSchedulePolicy, err := storkops.Instance().GetSchedulePolicy(migrationScheduleName)
+		if err != nil {
+			logrus.Errorf("Unable to get the created schedule policy")
+			return err
+		}
```

Comment:
nit.. We could use require.NoError. We could do this in several other places too. But here is where I noticed it and it's the new code being added so let's fix here?
`require.NoError(t, err, "Error getting created schedule policy")`

Reply:
Makes sense. Will address